openmdao/docs/features/model_visualization/n2_details.rst

```diff
@@ -79,183 +84,316 @@ In the upper triangular, the marked cells represent downstream connections. The
+   | |font_size|           | Font Size                       | Set the font size for the text in the diagram                                                      |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |collapse_depth|      | Set collapse depth              | Set how many levels are shown in the System hierarchy on the left                                  |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |model_height|        | Model Height                    | Set the height of the diagram in pixels                                                            |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |save_svg|            | Save SVG                        | Save the current view of the diagram to an SVG file                                                |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |show_node_info|      | Show / Hide Node Info           | Turning this on while result in information about the various nodes to be displayed while hovering |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |help|                | Help                            | Brings up the help window                                                                          |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+
+|
+
+Show Legend
+-----------
+If you click on the legend button |show_legend|, you will see the legend appear below the N2 diagram.
+The Legend window can then be dragged to a location of the user's choice.
 
+.. image::
+    images/legend.png
+    :align: center
+
+
+Here are explanations of the sections in the legend:
+
+    * The System & Variables section shows the colors for the different items in the model hierarchy.
+
+    * The N2 Symbols section shows the icons used to indicate the type of connection in the connection matrix.
+      The shape of the
+      icon in the matrix shows whether the connection is between scalars, vectors, or groups. The color of the icon is based
+      on the color associated with the variable of that type as shown in the System & Variables part of the legend.
+
+    * Finally, there is a section describing the colors for the items in the solver hierarchy on the right of the
```

Comment:
make parallel with the other bullets, e.g.: "The Solvers section shows the colors used for the different  solvers on the right side of the N2.  Either the Linear or Nonlinear solvers are shown depending on the state of the Toggle Solver Names button."